actions.csv

```diff
@@ -1,160 +1,175 @@
-100,2019/03/28,"https://github.com/996icu/996.icu/, https://radiichina.com/github-protest-chinese-tech-996/, https://www.ft.com/content/72754638-55d1-11e9-91f9-b6515a54c5b1",open_letter,working_conditions,"A campaign against long working hours by professionals in China's technology sector has become one of the fastest growing Github repository in Github's history. The ""anti-996"" campaign opposes employees working shifts of 9am-9pm, six days a week, which are commonly required by the country's tech giants.","online, china",None,241000,white_collar_workers,organizejs
-101,2019/04/01,"https://www.theregister.co.uk/2019/04/22/npm_fired_staff_union_complaints, https://www.salon.com/2019/04/30/software-engineers-at-prominent-tech-firm-say-they-were-fired-for-trying-to-unionize-report/",union_drive,working_conditions,"Recently fired NPM employees claim they were punished for trying to form a union. Four complaints have been filed with the National Labor Relations Board, alleging labor law violations by management.","oakland, california, usa",npm,3,white_collar_workers,bentarnoff
-102,2019/04/01,"https://www.theguardian.com/technology/2019/apr/01/google-kay-coles-james-removal-employees-letter, https://medium.com/@against.transphobia/googlers-against-transphobia-and-hate-b1b0a5dbf76",open_letter,"discrimination, ethics","A group of Google employees have called for the removal of a rightwing thinktank leader from the companies's new artificial intelligence council, citing her anti-LGBT and anti-immigrant record. Academic, civil society, and industry supporters have also signed the petition in mass.","online, usa",google,2556,"white_collar_workers, ai",organizejs
-103,2019/04/02,https://www.theguardian.com/technology/2019/apr/02/google-workers-sign-letter-temp-contractors-protest,open_letter,discrimination,"More than 900 Google workers have signed a letter objecting to the tech giant's treatment of temporary contractors, in what organizers are calling a ""historical coalition"" between Google""s full-time employees (FTEs) and temps, vendors and contractors (TVCs).","online, usa",google,900,"white_collar_workers, coworker_solidarity",organizejs
-104,2019/04/04,https://www.wired.com/story/microsoft-employees-protest-treatment-women-ceo-nadella/,protest,discrimination,"A group of Microsoft employees appeared at an employee meeting with CEO Satya Nadella Thursday to protest the companies's treatment of women. It's not clear how many were part of the protest, but some female and male employees at the event wore all white, inspired by the congresswomen who wore suffragette white to the State of the Union in February.","seattle, washington, usa",microsoft,None,"white_collar_workers, sexism",organizejs
-105,2019/04/10,"https://www.recode.net/2019/4/10/18304877/amazon-climate-change-employees-tech-activism, https://www.vox.com/business-and-finance/2019/4/11/18306389/amazon-employees-climate-change-plan",open_l
```

Comment:
foxxcon in brazil.. thats really interesting, had no idea.

Reply:
Yeah, quite unexpected!